README.md

```diff
@@ -32,6 +32,37 @@ oversubscription issues.
 
 ## Usage
 
+### Command Line Interface
```

Comment:
I think the result should be the same as in the `runtime introspection` section otherwise it might be confusing to see different libraries when using the cli and when using the function.

Reply:
Alright.